packages/manager/modules/pci/src/projects/project/data-processing/onboarding/translations/Messages_en_GB.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Translations must be scoped to their own AngularJS modules.

It means that you need to create a dedicated files for both:

- Messages_fr_FR.json
- Messages_en_GB.json

With their respective translations:

```json
{
  "data_processing_onboarding_content_1": "",
  "data_processing_onboarding_content_2": "",
  "data_processing_onboarding_content_3": "",
  …
}
```

```